README.md

```diff
@@ -1,2 +1,54 @@
-# comp110-worksheet-B
+# comp110-worksheet-B (SCOTT CALVERT)
 Base repository for COMP110 worksheet B
+
+![pict](https://github.com/Mindengine76/comp110-worksheet-B/blob/master/WIN_20171019_19_32_18_Pro.jpg)
+
+First level of abstraction
+
+1)select random word
+2)prompt the user that they only have a total of 5 guesses 
+3)prompt user to input guess
+
+4)match guess with randomly choosen word 
+
+5)if guess matches random word 
+	6)print "access granted"
+	
+7) if guess is incorrect then 
+	8) identify which letters are the same in both the guess and random word 
```

Comment:
The worksheet asked for pseudocode for the algorithm to calculate the number of letters correct, which you have not really done here. Be sure to read the specification carefully in future!